src/zrc/api/serializers/core.py

```diff
@@ -820,6 +820,13 @@ def save(self, **kwargs):
 
 
 class ZaakEigenschapSerializer(NestedHyperlinkedModelSerializer):
+    zaak = serializers.HyperlinkedRelatedField(
+        queryset=Zaak.objects.all(),
+        view_name="zaak-detail",
+        lookup_field="uuid",
+        validators=[IsImmutableValidator()],
+    )
+
```

Comment:
wouldn't we have achieved the same by specifying `"validators": [IsImmutableValidator()]` in the `extra_kwargs` for `zaak`?